.buildkite/libbeat/pipeline.libbeat.yml

```
@@ -9,6 +9,7 @@ env:
   GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
   IMAGE_UBUNTU_X86_64: "family/platform-ingest-beats-ubuntu-2204"
   IMAGE_WIN_2022: "family/platform-ingest-beats-windows-2022"
+  IMAGE_UBUNTU_X86_64_FIPS: "platform-ingest-beats-ubuntu-2204-fips-1752148343"
```

I think we can use the family instead, to get the latest versions instead:

```suggestion
  IMAGE_UBUNTU_X86_64_FIPS: "family/platform-ingest-beats-ubuntu-2204-fips"
```

---

This didn't work; I think it's because the FIPS image is on AWS and the `family/` prefix is only used for gcp?

---

@pazone , can you help here?

---

If I understand correctly, the intention is to use an AWS image with defined prefix. `imagePrefix: platform-ingest-beats-ubuntu-2204-fips`